proxy-node-gateway/src/test/java/uk/gov/ida/notification/apprule/EidasAuthnRequestAppRuleTests.java

```diff
@@ -166,6 +180,46 @@ public void serverErrorResponseFromVspLogsAndReturns500() throws Exception {
             .matches("Exception calling verify-service-provider for session '.*': Exception of type \\[REMOTE_SERVER_ERROR\\] whilst contacting uri:.*\n.*");
     }
 
+    @Test
+    public void checkIfHeadersIstioPersist() throws Exception {
+        final String xRequestId = "x-request-id";
```

Comment:
Rather than setting up these variables, can you just use the static fields from the top of the class?

Reply:
yeah can do 👍